tracecat/actions/core/email.py

```diff
@@ -86,7 +86,8 @@ def smtp_config(self):
             "auth": os.environ.get("SMTP_AUTH", "0"),
             "username": os.environ.get("SMTP_USER", ""),
             "password": os.environ.get("SMTP_PASS", ""),
-            "tls": os.environ.get("SMTP_TLS", "0"),
+            "ssl": os.environ.get("SMTP_SSL", "0"),
+            "tls": os.environ.get("SMTP_STARTTLS", "0"),
```

Comment:
nit: can we make `auth`, `ssl`, `tls`, `ignore_cert_errors` booleans? e.g.
```python
"ssl": os.environ.get("SMTP_SSL", "0").lower() in ("1", "true")
```

Reply:
also, might be cleaner if we also pull these configs out into `tracecat.config` so that we could reuse this for auth email verification too